### PR TITLE
Added extra controllers for storage

### PR DESCRIPTION
**Pull request by 3C111**
Now with more features:
- get a limited amount of buckets
- get all notebooks from a bucket

### CODE REVIEW

File: app/controllers/StorageExplorerController.scala [hunk 1]
```diff
@@ -89,43 +95,62 @@ class StorageExplorerController @Inject() (
...
+    nrfiles match {
+      case 0 =>
+        logger.debug( "No file limit requested, returning all files in bucket " )
+        g.V( Long.box( bucketid ) ).in( "resource:stored_in" ).in( "resource:has_location" ).has( Constants.TypeKey, "resource:file" )
 
-  def fileList( id: Long ): Action[AnyContent] = ProfileFilterAction( jwtVerifier.get ).async { implicit request =>
-    logger.debug( "Request to retrieve all files in bucket with id " + id )
+      case x: Int =>
+        logger.debug( "File limit of " + x + " requested, returning " + x + " file(s) in bucket " )
+        g.V( Long.box( bucketid ) ).in( "resource:stored_in" ).in( "resource:has_location" ).has( Constants.TypeKey, "resource:file" ).tail( x )
```
Comment (leafty) [inline app/controllers/StorageExplorerController.scala]: Limit, but no pagination?

**Comment by leafty** [comment]
How dare you improve code quality when asked to do maintenance of code :stuck_out_tongue_closed_eyes: ?

File: app/controllers/StorageExplorerController.scala (hunk 1 above)
Comment (3C111) [reply app/controllers/StorageExplorerController.scala]: Pagination upon request...if it is needed I'll put it

File: test/controllers/StorageExplorerControllerSpec.scala [hunk 2]
```diff
@@ -240,6 +268,8 @@ class StorageExplorerControllerSpec extends PlaySpec with OneAppPerSuite with Mo
       val content = contentAsJson( result ).as[Seq[PersistedVertex]]
       val contentTimeStamps = for ( node <- content ) yield node.properties.get( NamespaceAndName( "system", "creation_time" ) ).orNull.values.head.self
 
+      println( content )
```
Comment (ebiiii) [inline test/controllers/StorageExplorerControllerSpec.scala]: debug ?

Comment (3C111) [reply test/controllers/StorageExplorerControllerSpec.scala]: Thanks, removed it.

File: app/controllers/StorageExplorerController.scala (hunk 1 above)
Comment (ebiiii) [reply app/controllers/StorageExplorerController.scala]: As a maintenance fix, I don't think we will need it.

**Pull request merged**